core/src/main/java/greencity/controller/UserController.java

```diff
@@ -575,19 +578,87 @@ public void deleteHabit(
      */
     @ApiOperation(value = "Update user profile picture")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = HttpStatuses.OK),
-            @ApiResponse(code = 303, message = HttpStatuses.SEE_OTHER),
-            @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
-            @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 303, message = HttpStatuses.SEE_OTHER),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
     })
     @PatchMapping(path = "/profilePicture",
-            consumes = {MediaType.APPLICATION_JSON_UTF8_VALUE, MediaType.MULTIPART_FORM_DATA_VALUE})
+        consumes = {MediaType.APPLICATION_JSON_UTF8_VALUE, MediaType.MULTIPART_FORM_DATA_VALUE})
     public ResponseEntity<HttpStatus> updateUserProfilePicture(@ApiParam(value = "Profile picture")
                                                                @RequestPart(required = false) MultipartFile image,
                                                                @ApiIgnore
                                                                @AuthenticationPrincipal Principal principal) {
         String email = principal.getName();
-        userService.updateUserProfilePicture(image,email);
+        userService.updateUserProfilePicture(image, email);
         return ResponseEntity.status(HttpStatus.OK).build();
     }
+
+    /**
+     * Method for deleting user friend.
+     *
+     * @param friendId id user friend.
+     * @param userId   id current user.
+     * @author Marian Datsko
+     */
+    @ApiOperation(value = "Delete user friend")
+    @ApiResponses(value = {
```

Comment:
[optional] Why are you adding those `ApiResponses `with standard messages? 
It makes sense to add them if you want inform user of something with the `message` param.
I know that 200 - it's OK, that 400 - it's bad request. Those additional lines of code doesn't give any additional value.